kubectl-plugin/config/migrate.go

```diff
@@ -54,6 +55,18 @@ func (o *migrateOption) runE(cmd *cobra.Command, args []string) (err error) {
 		return
 	}
 
+	patchData := make(map[string]interface{})
+
+	kubesphereConfig, _, err := o.getKubeSphereConfig("kubesphere-config", "kubesphere-system")
+	if err != nil {
+		return err
+	}
+	if sonarQube, found, err := unstructured.NestedMap(kubesphereConfig, "sonarQube"); err != nil {
```

Comment:
`unstructured.NestedMap(kubesphereConfig, "sonarQube")` or `unstructured.NestedMap(kubesphereConfig, "devops.sonarqube")`

Reply:
I didn't see the difference about your suggestion.